bot/resources/tags/docstring.md

```diff
@@ -0,0 +1,16 @@
+A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that often used in file, classes, functions, etc. Docstrings usually has clear explanation, parameter(s) and return type.
```

Comment:
We should also comment on best practices in regards to what content should be in the explanation.